.github/workflows/jekyll.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy Website using Jekyll
+
+on:
+  push:
+    branches: ["master"]
```

Comment:
Shouldn't we run the `build` job on PRs to? And only the `deploy` job is restricted to master?